solr/core/src/java/org/apache/solr/cloud/api/collections/CreateCollectionCmd.java

```diff
@@ -236,21 +235,19 @@ public void call(ClusterState clusterState, ZkNodeProps message, NamedList resul
         }
 
         String baseUrl = zkStateReader.getBaseUrlForNodeName(nodeName);
-        //in the new mode, create the replica in clusterstate prior to creating the core.
+        //create the replica in clusterstate (i.e. ZK) prior to creating the core.
```

Comment:
s/clusterstate/state?

Reply:
"clusterstate" is used all over (including in the docs) to refer in general to the state of the cluster or collection stored in ZK.
Will replace with state.json here.

Reply:
I think we still can use 'clusterstate' as a generic term? When I read it in a sentence like this I do not read it as the file clusterstate.json, but as the cluster state. Perhaps we could consistenly write it as "cluster state" instead?

Reply:
Agreed, we should rename in many places. Although here state.json is appropriate since that's what happens.